### PR TITLE
Org table scroll

### PR DESCRIPTION
**Pull request by Stav-Kyriacou**

### CODE REVIEW

File: FE/src/app/components/organisation/organisation.component.scss [hunk 1]
```diff
@@ -43,6 +43,19 @@ button {
     margin-right: 10px;
 }
```
Comment (dan933) [inline FE/src/app/components/organisation/organisation.component.scss:45]: Adding

`    .flex-container {
        flex-direction: column;
        height: 442px;
    }`
    to `@media screen and (max-width:700px)`
    will allow scroll when the flex-direction is a column if we wanted to do that as well?

**Review by dan933 — APPROVED** [review]
Nice Stav that looks good :)

File: FE/src/app/components/organisation/organisation.component.scss (hunk 1 above)
Comment (Stav-Kyriacou) [reply FE/src/app/components/organisation/organisation.component.scss:45]: good point, i missed that

**Review by dan933 — APPROVED** [review]
Awesome nice fix

**Review by rikichoi — APPROVED** [review]

**Pull request merged**